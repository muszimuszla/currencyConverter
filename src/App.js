import React from "react";
import ExchangeRate from "./components/ExchangeRate";
import TransactionTop from "./components/TransactionsTop";
import TransactionsAll from "./components/TransactionsAll";

function App() {
  return (
    <React.Fragment>
      <div className="main-container">
        <div className="main-container__left">
          <ExchangeRate />
          <TransactionTop />
        </div>
        <div className="main-container__right">
          <TransactionsAll />
        </div>
      </div>
    </React.Fragment>
  );
}

export default App;
