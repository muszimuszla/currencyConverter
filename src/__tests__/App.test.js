import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("Main page", () => {
  it("renders without crashing", () => {
    shallow(<App />);
  });

  it("renders correctly", () => {
    const enzymeWrapper = shallow(<App />);
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
