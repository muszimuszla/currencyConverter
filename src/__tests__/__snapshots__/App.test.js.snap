// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page renders correctly 1`] = `
<Fragment>
  <div
    className="main-container"
  >
    <div
      className="main-container__left"
    >
      <Connect(ExchangeRate) />
      <Connect(TransactionsTop) />
    </div>
    <div
      className="main-container__right"
    >
      <Connect(TransactionsAll) />
    </div>
  </div>
</Fragment>
`;
